pkg/proxy/handlers/spacelister_list_test.go

```diff
@@ -22,127 +22,207 @@ import (
 	"github.com/codeready-toolchain/registration-service/pkg/proxy/metrics"
 	"github.com/codeready-toolchain/registration-service/pkg/signup"
 	"github.com/codeready-toolchain/registration-service/test/fake"
+	"github.com/codeready-toolchain/toolchain-common/pkg/test"
 )
 
-func TestSpaceListerList(t *testing.T) {
-	fakeSignupService, fakeClient := buildSpaceListerFakes(t)
-
-	t.Run("HandleSpaceListRequest", func(t *testing.T) {
-		// given
-		tests := map[string]struct {
-			username             string
-			expectedWs           []toolchainv1alpha1.Workspace
-			expectedErr          string
-			expectedErrCode      int
-			expectedWorkspace    string
-			overrideSignupFunc   func(ctx *gin.Context, userID, username string, checkUserSignupComplete bool) (*signup.Signup, error)
-			overrideInformerFunc func() service.InformerService
-		}{
-			"dancelover lists spaces": {
-				username: "dance.lover",
-				expectedWs: []toolchainv1alpha1.Workspace{
+func TestSpaceListerListUserWorkspaces(t *testing.T) {
```

Comment:
should we name it something like `TestListUserWorkspaces` since it's invoking the `ListUserWorkspaces` function ?